homeassistant/components/fritzbox/__init__.py

```diff
@@ -43,7 +44,15 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         CONF_CONNECTIONS: fritz,
     }
 
-    coordinator = FritzboxDataUpdateCoordinator(hass, entry)
+    try:
+        await hass.async_add_executor_job(fritz.update_templates)
+        LOGGER.debug("Enable smarthome templates")
```

Comment:
Please only wrap the line that can raise in the try... except block. Use an `else:` block if needed.